admin/resources/js/stories/TextArea.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import Form from "../components/form/Form";
+import Submit from "../components/form/Submit";
+import TextArea, { TextAreaProps } from "../components/form/TextArea";
+
+export default {
+  component: TextArea,
+  title: "Form/TextArea",
+} as Meta;
+
+const TemplateTextArea: Story<TextAreaProps> = (args) => {
+  return (
+    <div style={{ width: "20rem" }}>
```

Comment:
Is the wrapping div with a 20 rem width required here? It means that on a small mobile width, the element will overflow outside of the viewport.

Reply:
Answer to same question above.